gulpfile.js

```diff
@@ -20,23 +20,11 @@ var fs = require('fs');
 var yargs = require('yargs');
 var argv = yargs
   .usage('Usage: $0 <command> [<options>]')
-  .example('$0 build --minify=True')
-  .command('build', 'generate optimimized third party library for production',
-    function(yargs) {
-      argv = yargs
-        .usage('Usage: $0 build [--minify] [--output_directory]')
-        .option('minify', {
-          describe: 'Whether to minify third-party dependencies'
-        })
-        .option('output_directory', {
-          describe: 'A path to the directory where the files will be generated'
-        }).argv;
-    })
   .command('start_devserver', 'start GAE development server',
```

Comment:
Just to check -- if gulp is only meant as a background watcher for local files, why does it need all the start_devserver commands?

Reply:
Yes, sorry I forgot to mention that.

I discovered that in order to watch the files I need to start the server through gulp. I don't think there are any big drawbacks. If we replace the gulp watch with something else later it shouldn't be a big problem, because the start_devserver is mostly "piping" the data between Python and shell.

Also the main problem with gulp and build.py was about calling gulp from build.py and that was solved by this.

Reply:
OK -- can you add this info to the file for future reference? We want to establish a pattern for devs for when to use the gulpfile and when not to, so any exceptions should be called out explicitly and explained within the code. You don't need to talk about what happened in the past, just explain (at the top of the file) the overall vision of this file being just for watching changes to files and, at this point in the file, call out why there's an exception needed for start_devserver.

Reply:
Done.